packages/sit-onyx/src/components/OnyxLoadingIndicator/OnyxLoadingIndicator.stories.ts

```diff
@@ -21,10 +21,14 @@ const meta: Meta<typeof OnyxLoadingIndicator> = {
 export default meta;
 type Story = StoryObj<typeof OnyxLoadingIndicator>;
 
-export const Circle = { args: {} } satisfies Story;
-
 export const Dots = {
   args: {
     type: "dots",
   },
 } satisfies Story;
```

Comment:
```suggestion
export const Dots = { args: {} } satisfies Story;
```

Reply:
Yes, but when you for example choose "circle" in the controls and then reset the controls it does not switch back to "dots" and when you first load the page, the dots are not automatically selected in the controls. Thats why I changed it or is it suppose to work that way ?

Reply:
Ah okay then thats a Storybook bug :) Its fine like that